lib/active_admin/resource/attributes.rb

```diff
@@ -37,7 +37,10 @@ def sti_col?(c)
       end
 
       def counter_cache_col?(c)
-        c.name.end_with?("_count")
+        counter_cache_columns = resource_class.reflect_on_all_associations(:has_many)
+                                              .select(&:has_cached_counter?)
```

Comment:
@plashchynski thanks for contribution 👍 , I suggest to add new column  `some_count` to Post model in specs, and check it is rendered in features/index/format_as_csv.feature in order to merge it

Reply:
Are you still interested @plashchynski?